cnf-tests/TESTLIST.md

```diff
@@ -9,6 +9,7 @@ hack/fill-empty-docs.sh, check the json description files and fill the missing d
 The validation tests are preliminary tests intended to verify that the instrumented features are available on the cluster.
 | Test Name | Description |
 | -- | ----------- |
+|  | XXXXXXXX | 
```

Comment:
this doesn't look right (there are other instances below)

Reply:
This is caused by the new behavior of ginkgo's JUnit reporter, I contributed a fix already to ginkgo and it is going to be released around next week, I'll update the ginkgo's minor version then.

Reply:
ok, I guess we can move on then.

Reply:
oh
I see now it's already out

Reply:
cool, so let's prepare a followup PR to bump the dep and clean up this leftover